blocks/layout-grid/src/icons.js

```diff
@@ -5,16 +5,17 @@
 import { Path, SVG } from '@wordpress/components';
 
 export const GridIcon = ( props ) => {
+	const iconProps = {...props};
```

Comment:
Minor code style change
```suggestion
	const iconProps = { ...props };
```